apps/dcos_net/src/dcos_net_mesos_listener.erl

```diff
@@ -49,12 +49,16 @@
     timeout = 15000 :: timeout(),
     timeout_ref = make_ref() :: reference(),
```

Comment:
nit: you added few more timers to the state and this one is confusing, may I kindly ask you to rename it to `http_timeout` and `http_timeout_ref`?